src/components/DatePicker/DatePicker.jsx

```diff
@@ -40,10 +40,12 @@ export const DatePicker = ({
     timezoneName = null, // seller timezone (e.g. "America/Los_Angeles") to return correct today date
     ...rest
 }) => {
-    const initialValue = variant === variants.single ? value : value.from;
+    const initialValue = variant === variants.single ? value : value?.from;
```

Comment:
we have reverted your previous PR. can you check if this PR has all the changes that are required for the fix ?

Reply:
Also conflicts were resolved